### PR TITLE
Macro to run cljs tests without specifying all namespaces

### PR DESCRIPTION
**Pull request by pablodip**
Main two benefits:
* no test namespace is overlooked when running all tests
* no need to write every test namespace

Some features.

It allows to exclude some namespaces, as it is done in the runner right now with one namespace.

It also allows to sort some namespaces, since without that tests were failing with this:

```
Testing status-im.test.utils.async
undefined
/target/test/status_im/utils/utils.cljs:61
(defn http-get
^
ReferenceError: window is not defined
    at Function.status_im.utils.utils.http_get.cljs$core$IFn$_invoke$arity$4 (/Users/pablodip/Code/testing/status-react/target/test/status_im/utils/utils.cljs:61:1)
    at status_im$utils$utils$http_get (/Users/pablodip/Code/testing/status-react/target/test/status_im/utils/utils.cljs:61:1)
    at Function.status_im.utils.utils.http_get.cljs$core$IFn$_invoke$arity$3 (/Users/pablodip/Code/testing/status-react/target/test/status_im/utils/utils.cljs:63:5)
    at status_im$utils$utils$http_get (/Users/pablodip/Code/testing/status-react/target/test/status_im/utils/utils.cljs:61:1)
    at status_im$utils$prices$get_prices (/Users/pablodip/Code/testing/status-react/target/test/status_im/utils/prices.cljs:28:4)
    at /Users/pablodip/Code/testing/status-react/target/test/status_im/ui/screens/wallet/events.cljs:54:6
    at re_frame$fx$do_fx_after (/Users/pablodip/Code/testing/status-react/target/test/re_frame/fx.cljc:72:16)
    at re_frame$interceptor$invoke_interceptor_fn (/Users/pablodip/Code/testing/status-react/target/test/re_frame/interceptor.cljc:67:6)
    at re_frame$interceptor$invoke_interceptors (/Users/pablodip/Code/testing/status-react/target/test/re_frame/interceptor.cljc:105:23)
    at re_frame$interceptor$execute (/Users/pablodip/Code/testing/status-react/target/test/re_frame/interceptor.cljc:198:8)
Error encountered performing task 'doo' with profile(s): 'test'
```

Since it was failing in the namespace `status-im.test.utils.async`, I just sorted that, and the rest of the namespaces are with no specific order. Feel free to tell me the logic with which the namespaces should be sorted and I will improve that sorting logic.

### CODE REVIEW

**Comment by pablanopete** [comment]
Hi @pablodip and thanks for the contribution! In general, we prefer pull requests that solve specific issues that have been identified and agreed upon. This way we can discuss trade-offs about potential solutions before implementing them. If you experience a specific pain point, feel free to open an issue with this.

Feel free to reach out to me Hutch, at https://chat.status.im/#/register if you’d like about further contributions. Thanks! 🎉

**Comment by pablodip** [comment]
Thanks, @pablanopete, will have that in mind. :)

File: test/cljs/status_im/test/runner_macros.clj [hunk 1]
```diff
@@ -0,0 +1,22 @@
+(ns status-im.test.runner-macros
+  (:require [cljs.analyzer.api :as ana-api]))
+
+(def cljs-tests-ordered-ns (->> ['status-im.test.utils.async]
```
Comment (yenda) [inline test/cljs/status_im/test/runner_macros.clj:4]: why is this one left explicitely ?

**Comment by yenda** [comment]
Interesting series of PRs, nice to see propositions of improvements that we might not have think off. Thank you for your contributions !

File: test/cljs/status_im/test/runner_macros.clj (hunk 1 above)
Comment (pablodip) [reply test/cljs/status_im/test/runner_macros.clj:4]: Tests were failing with random execution order (I pasted the error into the PR), so I tried to reproduce the execution order that was working in the current runner. I don't know why the order is needed though, so if anyone has any clue, that could help detail why and maybe improve the sorting logic, or even get rid of the order requirement for tests to pass.

**Comment by oskarth** [comment]
Thanks for the PR! Unfortunately, I'm going to close this as it doesn't appear to solve an identified problem. As far as I'm aware, this is not a big pain point, and it also introduces significant complexity with macros that might cause unforseen issues in the future.

In general, we strongly prefer PRs that address already created issues. You are welcome to create an issue describing the problem in detail if you feel like this is in error. Don't hesitate to reach out to us on Riot as well :)

**Pull request closed without merging**